src/main/content/_assets/js/builds.js

```diff
@@ -25,6 +25,11 @@ function render_builds(builds, parent) {
     var analytics_class_name = 'link_' + parent.parent().data('builds-id');
     var download_arrow = '<div class="download_arrow"><div class="table_arrow"></div><div class="table_line"></div></div>';
 
+    // update maven and gradle commands to use latest version
+    if (parent.parent().data('builds-id') == "runtime_releases") {
```

Comment:
Lets be more paranoid here and do some checking, like that the returned version contains three periods `.` or you can test further that it's [integer].[integer].[integer].[integer]